rules/assign-author.ts

```diff
@@ -0,0 +1,21 @@
+import { danger, schedule } from "danger";
+
+// Make schedule testable with Jest. Inspiration: https://git.io/fNh6i
+const testableSchedule = (reason: string, action: any) =>
+  typeof jest !== "undefined" ? action : schedule(action);
+
+export const assignAuthor = testableSchedule(
+  "Automatically assign the author to a new pull request",
+  async () => {
+    const github = danger.github as any;
+    const thisPR = github.thisPR;
+    const pr = github.pr;
+
+    github.api.issues.addAssigneesToIssue({
+      assignees: pr.user,
```

Comment:
I'm super sleepy and am not great with TypeScript but I have a feeling this should be an Array?